scripts/release_scripts/update_changelog_and_credits.py

```diff
@@ -459,22 +460,49 @@ def main():
     github_username = parsed_args.github_username
 
     personal_access_token = common.get_personal_access_token()
+
+    python_utils.PRINT('Generating release summary...')
+    release_info.main(personal_access_token)
+
+    if not os.path.exists(release_constants.RELEASE_SUMMARY_FILEPATH):
+        raise Exception(
+            'Release summary file %s is missing. Please re-run '
+            'this script.' % release_constants.RELEASE_SUMMARY_FILEPATH)
+
     g = github.Github(personal_access_token)
     repo_fork = g.get_repo('%s/oppia' % github_username)
 
-    current_release_version = branch_name[len(
-        common.RELEASE_BRANCH_NAME_PREFIX):]
+    current_release_version = branch_name[
+        len(common.RELEASE_BRANCH_NAME_PREFIX):]
     target_branch = 'update-changelog-for-releasev%s' % current_release_version
 
     remove_updates_and_delete_branch(repo_fork, target_branch)
 
-    message = (
-        'Please update %s to:\n- have a correct changelog for '
-        'updating the CHANGELOG file\n- have a correct list of new '
-        'authors and contributors to update AUTHORS, CONTRIBUTORS '
-        'and developer_names section in about-page.directive.html\n' % (
+    # Opens Credit Form.
+    python_utils.PRINT(
+        'Note: Make following changes directly to %s and make sure to '
+        'save the file after making these changes.' % (
             release_constants.RELEASE_SUMMARY_FILEPATH))
-    common.ask_user_to_confirm(message)
+
+    common.ask_user_to_confirm(
+        'Check emails and names for authors and contributors and '
+        'verify that the emails are '
+        'correct through welcome emails sent from welcome@oppia.org '
+        '(Confirm from Sean in case of doubt).')
+    common.open_new_tab_in_browser_if_possible(
+        release_constants.CREDITS_FORM_URL)
+    common.ask_user_to_confirm(
+        'Check the credits form and add any non technical contributors '
+        'to the contributor list in release summary file.')
+    common.ask_user_to_confirm(
+        'Categorize the terms in uncategorized section and arrange '
```

Comment:
"terms" sounds like it refers to "terms and conditions" pages.

Let's make it "Categorize the PR titles in the Uncategorized section of the changelog, and arrange the changelog to have user-facing categories on top."

Reply:
Done.